test/e2e/specs/delete-prohibited/recordset/histogram-facet.spec.ts

```diff
@@ -0,0 +1,348 @@
+import { test, expect, Locator } from '@playwright/test';
+
+// locators
+import RecordsetLocators, { TimestampDateTime } from '@isrd-isi-edu/chaise/test/e2e/locators/recordset';
+
+// utils
+import { getCatalogID } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
+import { 
+  testRangePickerInputsAfterZoom, testTimestampRangePickerInputsAfterZoom 
+} from '@isrd-isi-edu/chaise/test/e2e/utils/recordset-utils';
+
+const testParams = {
+  schema_name: 'histogram-faceting',
+  table_name: 'main',
+  totalNumFacets: 4,
+  facetNames: ['int_col', 'float_col', 'date_col', 'timestamp_col'],
+  facets: [
+    {
+      name: 'int_col',
+      initialMin: '1',
+      initialMax: '30',
+      zoom1: { min: '7', max: '25' },
+      zoom2: { min: '9', max: '21' },
+      allRecords: 'Displaying first25of 195 matching results',
+      zoom3: { min: '11', max: '19' }
+    }, {
+      name: 'float_col',
+      initialMin: '1.10',
+      initialMax: '30.30',
+      zoom1: { min: '6.94', max: '24.46' },
+      zoom2: { min: '10.44', max: '20.96' },
+      allRecords: 'Displaying first25of 155 matching results',
+      zoom3: { min: '12.55', max: '18.85' }
+    }, {
+      name: 'date_col',
+      initialMin: '2001-01-01',
+      initialMax: '2030-06-30',
+      zoom1: { min: '2006-11-27', max: '2024-08-12' },
+      zoom2: { min: '2010-06-13', max: '2021-01-28' },
+      allRecords: 'Displaying first25of 165 matching results',
+      zoom3: { min: '2012-07-30', max: '2018-12-20' }
+    }, {
+      name: 'timestamp_col',
+      initialMin: { date: '2007-04-06', time: '01:01:01' },
+      initialMax: { date: '2007-09-30', time: '06:30:31' },
+      zoom1: {
+        min: { date: '2007-05-11', time: '11:42:55' },
+        max: { date: '2007-08-25', time: '19:48:37' }
+      },
+      zoom2: {
+        min: { date: '2007-06-01', time: '18:08:03' },
+        max: { date: '2007-08-04', time: '13:23:28' }
+      },
+      allRecords: 'Displaying first25of 155 matching results',
+      zoom3: {
+        min: { date: '2007-06-14', time: '12:23:08' },
+        max: { date: '2007-07-22', time: '19:08:23' }
+      }
+    }
+  ]
+};
+
+test.describe('Testing features for range picker facet types with histograms', () => {
+  test.describe.configure({ mode: 'parallel' });
+
+  for (const [index, facetParams] of testParams.facets.entries()) {
```

Comment:
This line started causing a compilation error in the editor but still ran the tests regardless. The associated change was to bump the target version of ecmascript in tsconfig to `es6` which I think should be fine since it's been fully supported since 2017.

I'm unsure why this started happening here and not the other specs that I wrote using `for of` with `entries`.